### ISSUE TITLE
lldb doesn't work in Swift

### ISSUE THREAD

**Issue by adek05** [AFNetworking/AFNetworking#2550]
I propose this codemod:

```
codemod.py -m -d . --extensions h '@property \(nonatomic, strong\) dispatch_' '@property (nonatomic, assign) dispatch_'
```

These are the errors I get.
http://pastebin.com/Jd6UPWXL

cc @gricha

**Comment by Gricha** [comment]
Hey, me and @adek05 looked into the issue and error message and went to conclusion that there is no reason in defining property as strong referenced if it is a C type. From what I know it is basically OK to define those properties as assign and they will be retained in ARC. Analyzer doesn't show any leaks so this seems to be all right. 

Are you ok with our codemod? We can pull request any time.

**Comment by mattt** [comment]
> ...there is no reason in defining property as strong referenced if it is a C type.

GCD datatypes are treated as objects by ARC if `OS_OBJECT_USE_OBJC` is defined, which is true as of iOS 6 / OS X 10.8. For these base SDKs, `strong` is the only appropriate ownership for properties.

If LLDB is incorrectly operating under incompatible settings, that's an issue with Xcode configuration or LLDB itself, and should be addressed as such. There is nothing to change in AFNetworking.

**Comment by Gricha** [comment]
Matt, thank you for clarification. I agree. I do not see any problems coming from lldb itself as it works for pure Objective-C, but not with Swift, I'm blaming Xcode.

**Comment by mattt** [comment]
Fixed by 3a1785c.

**Comment by Gricha** [comment]
Thanks @mattt, it is awesome!
